input-output-hk/lace
# [LW-9499] New wallet manager migration

# Checklist

- [X] JIRA - https://input-output.atlassian.net/browse/LW-9499

---

## Proposed solution

Creates migration for newwallet manager

## Testing

Unlocked wallet test

1) Install the extension with main branch
2) Create/import new wallet
3) Build this branch, make sure the version in the manifest file is higher then that of the main branch (you might have to manually modify it)
4) Install the extension with the new build
5) Open lace and make sure it is working properly (to make sure the migration ran, verify that lock and keyAgentData are not present in localStorage)

Locked wallet test
1) Repeat the process but lock the wallet after step 2. 

## Screenshots

Attach screenshots here if implementation involves some UI changes
